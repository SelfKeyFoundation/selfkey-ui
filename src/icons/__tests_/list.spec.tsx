import * as React from 'react';
import { create } from 'react-test-renderer';
import ListIcon from '../list';

describe('List Icon', () => {
  it('should show List Icon', () => {
    const component = create(
      <ListIcon />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});