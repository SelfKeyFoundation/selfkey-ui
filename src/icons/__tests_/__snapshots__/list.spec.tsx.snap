// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Icon should show List Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root"
  focusable="false"
  style={
    Object {
      "height": "16px",
      "width": "32px",
    }
  }
  viewBox="0 0 32 16"
>
  <g
    fillRule="evenodd"
    id="ðŸŒˆ-UI-Guidelines"
    stroke="none"
    strokeWidth="1"
  >
    <g
      fill="inherit"
      id="3.-Style-Guide---Forms"
      transform="translate(-1098.000000, -863.000000)"
    >
      <g
        id="Line-3-Buttons"
        transform="translate(39.000000, 559.000000)"
      >
        <g
          id="button-groups"
          transform="translate(979.000000, 242.000000)"
        >
          <g
            id="Button-group"
            transform="translate(7.000000, 48.000000)"
          >
            <g
              id="button-list"
              transform="translate(59.000000, 0.000000)"
            >
              <path
                d="M14,30 L17.2,30 L17.2,26.8 L14,26.8 L14,30 Z M14,17.2096 L17.2,17.2096 L17.2,14.0096 L14,14.0096 L14,17.2096 Z M14,23.6 L17.2,23.6 L17.2,20.4 L14,20.4 L14,23.6 Z M20.4,30 L46,30 L46,26.8 L20.4,26.8 L20.4,30 Z M20.4,17.2 L46,17.2 L46,14 L20.4,14 L20.4,17.2 Z M20.4,23.6 L46,23.6 L46,20.4 L20.4,20.4 L20.4,23.6 Z"
                id="icon-list"
              />
            </g>
          </g>
        </g>
      </g>
    </g>
  </g>
</svg>
`;
